### PR TITLE
Added agent and 4.19 jobs

### PR DESCRIPTION
**Pull request by KeerthanaAP**

### CODE REVIEW

File: p_periodic.json [hunk 1]
```diff
@@ -1,6 +1,4 @@
 {
-    "4.11 to 4.12 upgrade": "periodic-ci-openshift-multiarch-master-nightly-4.12-upgrade-from-nightly-4.11-ocp-ovn-remote-libvirt-ppc64le",
-    "4.12 libvirt": "periodic-ci-openshift-multiarch-master-nightly-4.12-ocp-e2e-ovn-remote-libvirt-ppc64le",
     "4.12 to 4.13 upgrade": "periodic-ci-openshift-multiarch-master-nightly-4.13-upgrade-from-nightly-4.12-ocp-ovn-remote-libvirt-ppc64le",
     "4.12 heavy build": "periodic-ci-openshift-multiarch-master-nightly-4.12-ocp-heavy-build-ovn-remote-libvirt-ppc64le",
```
Comment (Shilpa-Gokul) [inline p_periodic.json:3]: This can also be removed

Comment (Shilpa-Gokul) [reply p_periodic.json:3]: Also 4.12 to 4.13 upgrade

Comment (KeerthanaAP) [reply p_periodic.json]: Done.

**Pull request merged**